apps/emqx_bridge/src/emqx_bridge_api.erl

```diff
@@ -395,7 +400,7 @@ schema("/bridges/:id/enable/:enable") ->
                 responses =>
                     #{
                         204 => <<"Success">>,
-                        400 => error_schema('INVALID_ID', "Bad bridge ID"),
+                        404 => error_schema('NOT_FOUND', "Bridge not found or operation invalid"),
```

Comment:
```suggestion
                        404 => error_schema('NOT_FOUND', "Bridge not found or invalid operation"),
```
And similar for a few occurrences below.

This would also make it consistent with l. 636.